doc/posts/2023/2023_05_19_vara_week0.rst

```diff
@@ -0,0 +1,34 @@
+GSOC 2023 week 0 blog
+
+.. post:: May 19 2023
+     :author: Vara Lakshmi Bayanagari
+     :tags: google
+     :category: gsoc
+
+While applying for the GSOC 2023 DIPY sub-project titled “Creating Synthetic MRI”, I knew 
```

Comment:
Please, add subsection title, it helps

Reply:
Hi @skoudoro,
I didn't quite understand the subsection part. Can you explain?